packages/contracts-bedrock/src/L1/OptimismPortalInterop.sol

```diff
@@ -40,7 +40,7 @@ contract OptimismPortalInterop is OptimismPortal {
                 uint256(0), // value
                 uint64(SYSTEM_DEPOSIT_GAS_LIMIT), // gasLimit
                 false, // isCreation,
-                abi.encodeCall(L1BlockInterop.setConfig, (_type, _value))
+                abi.encodeCall(L1BlockIsthmus.setConfig, (_type, _value))
```

Comment:
@skeletor_spaceman I only updated this on this contract, bc otherwise it will fail due to a different function name.
But should I update the contract version bumping to a patch?